docs/dev/developer-guides/aa.md

```diff
@@ -18,12 +15,9 @@ As a result, such applications require L1 relayers, e.g. an EOA to help facilita
 Accounts in zkSync Era can initiate transactions, like an EOA, but can also have arbitrary logic implemented in them, like a smart contract. This feature is called "account
 abstraction" and it aims to resolve the issues described above.
 
-::: warning Unstable feature
-
-This is the test release of account abstraction (AA) on zkSync Era. We are very happy to hear your feedback! Please note: **breaking changes to the API/interfaces required for AA should be anticipated.**
-
-zkSync Era is one of the first EVM-compatible chains to adopt AA, so this testnet is also used to see how "classical" projects from EVM chains can coexist with the account abstraction feature.
-
+::: warning
+- zkSync Era is one of the first EVM-compatible chains to adopt AA, so we are using the testnet to see how *classical* projects from EVM chains coexist with the account abstraction feature.
```

Comment:
what is AA? (it's not defined).

", so we are using" --> ". We are using" ?  (it just sounds off).

"with the account abstraction feature" --> "with account abstraction" ?